clang/test/Parser/cxx03-lambda-extension.cpp

```diff
@@ -0,0 +1,5 @@
+// RUN: %clang_cc1 -fsyntax-only -Wno-unused-value -verify -std=c++03 %s
+
+void func() {
```

Comment:
We should also test that `__has_feature(cxx_lambdas)` produces the correct results -- I think you're missing changes in `include/clang/Basic/Features.def`.